pkg/services/accesscontrol/checker.go

```diff
@@ -0,0 +1,57 @@
+package accesscontrol
+
+import (
+	"github.com/grafana/grafana/pkg/services/user"
+)
+
+func Checker(user *user.SignedInUser, action string) func(scopes ...string) bool {
```

Comment:
nit: what is the checking for?

I read check actions. this is a potential renaming scenario further down. so really nit

Reply:
It constructs a list of dashboard/folder UIDs that the user has access to, and returns a function that checks if the given UIDs are in this list.